### PR TITLE
Code Coverage

### PR DESCRIPTION
**Pull request by benhowes**
### What is the context of this PR?

This PR increases code coverage of the project to 100%. There are no logic changes.

**isnt this just a vanity metric?**: Yes, however it means that unrelated PR's which remove lines of code can fail the code-cov checks

**Oh yeah, like when?**: If you have a file where 98/100 (98%) lines are covered, then you remove 2  lines which (had coverage) by refactoring, but don't make changes affecting the untested lines, then you have 96/98 (97.9%) coverage. Reducing coverage in a PR is not acceptable from a PR.

**How come not only test code has changed then?**: I have had to make a few small refactors to make patching or testing easier - There should be no logic changes!

Where ever I've tested something, I've tried to ensure that I am adding tests which will be useful, rather than just getting coverage for the sake of it! e.g. QuestionnaireStore had really high coverage, but no unit tests.

### How to review 
Everything should be exactly the same as before, test a few surveys, dumping data etc.

### CODE REVIEW

File: app/setup.py [hunk 1]
```diff
@@ -178,27 +178,28 @@ def setup_secure_headers(application):
         frame_options='DENY')
 
 
-def setup_database(application):
-    application.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+def get_database_uri(application):
+    """ Returns database URI. Prefer SQLALCHEMY_DATABASE_URI over components."""
```
Comment (insacuri) [inline app/setup.py:182]: I love it!

**Review by kierenbeckett — APPROVED** [review]

File: app/data_model/questionnaire_store.py [hunk 2]
```diff
@@ -21,6 +22,9 @@ def __init__(self, storage, version=LATEST_VERSION):
         if version is not None:
             self.version = version
```
Comment (dcdarrell9) [inline app/data_model/questionnaire_store.py]: Do we need this?

**Review by dcdarrell9 — APPROVED** [review]

**Comment by MebinAbraham** [comment]
@benhowes Squash and rebase?

**Comment by benhowes** [comment]
@MebinAbraham will do, I've been trying to get master sorted out first. We're nearly there now.

**Pull request merged**